packages/react-devtools-extensions/src/backend.js

```diff
@@ -7,6 +7,10 @@
 'use strict';
 
 let welcomeHasInitialized = false;
+const _window = {
```

Comment:
Do we have a naming conversation about variables beginning with `_`? In some other projects, those variables means "not used"